presto-hive/src/main/java/io/prestosql/plugin/hive/HiveTableProperties.java

```diff
@@ -129,8 +129,8 @@ public HiveTableProperties(TypeManager typeManager, HiveConfig config)
                         false),
                 integerProperty(BUCKET_COUNT_PROPERTY, "Number of buckets", 0, false),
                 stringProperty(AVRO_SCHEMA_URL, "URI pointing to Avro schema for the table", null, false),
-                integerProperty(TEXTFILE_SKIP_HEADER_LINE_COUNT, "Number of header lines", null, false),
-                integerProperty(TEXTFILE_SKIP_FOOTER_LINE_COUNT, "Number of footer lines", null, false),
+                integerProperty(SKIP_HEADER_LINE_COUNT, "Number of header lines", null, false),
```

Comment:
@electrum @findepi do you guys like his rename of `textfile_skip_header_line_count` to `skip_footer_line_count` hive table property?